scripted-tests/run/execution-context/src/main/scala/Hello.scala

```diff
@@ -0,0 +1,7 @@
+object Test {
+  def main(args: Array[String]): Unit = {
+    import scala.concurrent.Future
+    import scala.concurrent.ExecutionContext.Implicits.global
+    Future(1 + 2).map(_ + 3).map(_ + 4).foreach(println)
+  }
```

Comment:
You should also `println("main")` at the end of `main()` (after the `Future(1 + 2)...` line) and assert that the output is `"main\n10"` (and not `"10\nmain"`).